src/status_im/ui/screens/profile/components/views.cljs

```diff
@@ -27,44 +28,56 @@
             :accessibility-label :username-input}
            props)]])
 
-(defn- show-profile-icon-actions [options]
-  (when (seq options)
-    (list-selection/show {:title   (i18n/label :t/image-source-title)
-                          :options options})))
-
-(defn- names [{:keys [name public-key] :as contact}]
-  (let [generated-name (when public-key (gfy/generate-gfy public-key))]
-    [react/view styles/profile-header-name-container
-     [react/text {:style           styles/profile-name-text
-                  :number-of-lines 1}
+(defn- names [{:keys [usernames name public-key] :as contact}]
+  (let [generated-name (when public-key (gfy/generate-gfy public-key))
+        with-sub?      (seq usernames)]
+    [react/view (if with-sub? styles/profile-header-name-container-with-subtitle
+                    styles/profile-header-name-container)
+     [react/text {:style (if with-sub? styles/profile-name-text-with-subtitle
+                             styles/profile-name-text)}
       (multiaccounts/displayed-name contact)]
-     (when generated-name
+     (when with-sub?
```

Comment:
what is `sub`?

Reply:
"with subtitle" 😅 bad naming sorry.. will spell it out
so, if there are ens user names... the text/title part of the profile header will have `three-word` name as subtitle